trinity/rpc/modules/eth.py

```diff
@@ -80,15 +91,43 @@ def get_block_at_number(chain: BaseChain, at_block: Union[str, int]) -> BaseBloc
         return chain.get_block_by_header(at_header)
 
 
+def dict_to_transaction(
```

Comment:
minor naming nitpick.  I think this function is specific to `call` style transactions.  Maybe we should namespace it as such since this isn't suitable for endpoints like `sendTransaction`

Reply:
Cool, I think `dict_to_spoof_transaction` should communicate that. You don't want a spoof transaction for sending, only for calls & gas estimates.